notebooks/exercise7-oqmd-optimade-tutorial.ipynb

```diff
@@ -434,10 +411,10 @@
     "# Load the data generated in previous sections\n",
     "\n",
     "# Input feature data - generated using Magpie on POSCAR files\n",
-    "if not os.path.isfile(\"features_processed_final.csv\"):\n",
-    "    raise RuntimeError(\"Please complete the optional step above, or copy the example data from `oqmd_exercise_data` into this directory.\")\n",
-    "\n",
-    "features = open(\"./features_processed_final.csv\",\"r\").read().strip().split(\"\\n\")\n",
+    "if os.path.isfile(\"./features_processed_final.csv\"): # Case where the optional part has been done\n",
+    "    features = open(\"./features_processed_final.csv\",\"r\").read().strip().split(\"\\n\")\n",
+    "else: \n",
+    "    features = open(\"./oqmd_exercise_data/features_processed_final.csv\",\"r\").read().strip().split(\"\\n\") # When the optional part was not done\n",
```

Comment:
Part of the problem here is that the students wont have access even to `./oqmd_exercise_data/` if they are running with Colab.

Reply:
Would it not be better to download the files from some location if it is so difficult to read the files from a hard drive in colab? 
The other option would be to drop colab. And just use binder or let the students run the jupyter notebook localy.